buildkitd/buildkitd.go

```diff
@@ -1014,7 +1014,7 @@ func getCacheSize(ctx context.Context, volumeName string, fe containerutil.Conta
 	return int(infos[volumeName].SizeBytes), nil
 }
 
-func addRequiredOpts(settings Settings, installationName string, isUsingPodman bool, opts ...client.ClientOpt) ([]client.ClientOpt, error) {
+func addRequiredOpts(settings Settings, _ string, _ bool, opts ...client.ClientOpt) ([]client.ClientOpt, error) {
```

Comment:
Same here and everywhere else (if any). Remove unused params and update callers, please.